### PR TITLE
Update Swagger UI version from 3.52.3 to 4.6.1

### PR DESCRIPTION
**Pull request by eduardoj**
Update the Swagger UI to the new released version.

https://github.com/swagger-api/swagger-ui/releases/tag/v4.6.1

### CODE REVIEW

**Comment by eduardoj** [comment]
To test the changes with the review app, go to: https://obs-reviewlab.opensuse.org/eduardoj-update_swagger_ui_to_v4_6_1/apidocs-new/index.html

**Review by saraycp — APPROVED** [review]

File: src/api/public/apidocs-new/oauth2-redirect.html [hunk 1]
```diff
@@ -67,9 +67,13 @@
         window.close();
     }
 
-    window.addEventListener('DOMContentLoaded', function () {
-      run();
-    });
+    if (document.readyState !== 'loading') {
```
Comment (dmarcoux) [inline src/api/public/apidocs-new/oauth2-redirect.html:70]: What is the reason for this change?

Comment (eduardoj) [reply src/api/public/apidocs-new/oauth2-redirect.html:70]: You may want to have look at the [upstream project](https://github.com/swagger-api/swagger-ui/commit/3eb05287efec9300ae612092e44f5378113179a3).

I documented how we update Swagger UI in [our wiki](https://github.com/openSUSE/open-build-service/wiki/New-Swagger-API-doc#how-to-update). Please have a look.

Comment (dmarcoux) [reply src/api/public/apidocs-new/oauth2-redirect.html:70]: Oh okay, this is from upstream :+1:

**Review by dmarcoux — APPROVED** [review]
